src/contracts/contract_deployer.star

```diff
@@ -1,12 +1,61 @@
 IMAGE = "ethpandaops/optimism-contract-deployer:latest"
 
 ENVRC_PATH = "/workspace/optimism/.envrc"
-
 FACTORY_DEPLOYER_ADDRESS = "0x3fAB184622Dc19b6109349B94811493BF2a45362"
+# raw tx data for deploying Create2Factory contract to L1
 FACTORY_DEPLOYER_CODE = "0xf8a58085174876e800830186a08080b853604580600e600039806000f350fe7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf31ba02222222222222222222222222222222222222222222222222222222222222222a02222222222222222222222222222222222222222222222222222222222222222"
 
+def deploy_factory_contract(
+    plan,
+    priv_key,
+    l1_config_env_vars,
+):
+    factory_deployment_result = plan.run_sh(
+        description="Deploying L2 factory contract to L1",
```

Comment:
This will take a longer time, as it needs a finalized L1 chain. Might worth adding that to the description.

Reply:
It takes 4 mins for minimal preset and 30 mins with mainnet preset. So bit more than a second.